src/main/java/net/openhft/hashing/LongHashFunction.java

```diff
@@ -290,6 +290,33 @@ public static LongHashFunction xx(long seed) {
         return XxHash.asLongHashFunctionWithSeed(seed);
     }
 
+    /**
```

Comment:
Please also add notes about the new algorithm to readme.md and package-info.java.